RS: TLS 1.3 (#3021)
RedisLabs/redislabs-docs: content/rs/references/rest-api/objects/cluster/_index.md

| cnm_http_port | integer, (range:&nbsp;1024-65535) | API HTTP listening port |
| cnm_https_port | integer, (range:&nbsp;1024-65535) | API HTTPS listening port |
| control_cipher_suites | string | Specifies the enabled ciphers for the control plane. The ciphers are specified in the format understood by the BoringSSL library. |

> | control_cipher_suites_tls_1_3 | string | Specifies the enabled TLS 1.3 ciphers for the control plane. The ciphers are specified in the format understood by the BoringSSL library. |

Reviewer: control_cipher_suites_tls_1_3 - add that it's read only

| reserved_ports | array of strings | List of reserved ports and/or port ranges to avoid using for database endpoints (for example <nobr>`"reserved_ports": ["11000", "13000-13010"]`</nobr>) |
| s3_url | string | Specifies the URL for S3 export and import |
| saslauthd_ldap_conf | string | saslauthd LDAP configuration |

> | sentinel_cipher_suites | array | Specifies the list of enabled ciphers for the sentinel service. The supported ciphers are ones that were implemented by the [cipher_suites.go](<https://golang.org/src/crypto/tls/cipher_suites.go>) package. |

Reviewer 2: ```suggestion
| sentinel_cipher_suites | array | Specifies the list of enabled ciphers for the sentinel service. The supported ciphers are those implemented by the [cipher_suites.go](<https://golang.org/src/crypto/tls/cipher_suites.go>) package. |
```

> | sentinel_cipher_suites_tls_1_3 | string | Specifies the list of enabled TLS 1.3 ciphers for the discovery (sentinel) service. The supported ciphers are those implemented by the [cipher_suites.go](<https://golang.org/src/crypto/tls/cipher_suites.go>) package. |

Reviewer: sentinel_cipher_suites_tls_1_3 - add that it's read only

> | sentinel_cipher_suites_tls_1_3 | string | Specifies the list of enabled TLS 1.3 ciphers for the discovery (sentinel) service. The supported ciphers are ones that were implemented by the [cipher_suites.go](<https://golang.org/src/crypto/tls/cipher_suites.go>) package. |

Reviewer 2: ```suggestion
| sentinel_cipher_suites_tls_1_3 | string | Specifies the list of enabled TLS 1.3 ciphers for the discovery (sentinel) service. The supported ciphers are those implemented by the [cipher_suites.go](<https://golang.org/src/crypto/tls/cipher_suites.go>) package. |
```
